### ISSUE TITLE
Why is it necessary to *delete* the config file for script usage?

### ISSUE THREAD

**Issue by irl** [aerth/go-quitter#18]
This could be better explained in the documentation, because I don't believe you.

**Comment by aerth** [comment]
Thanks! The default password was changed and you are using an older config version for a newer go-quitter.
The file should work if you line up the versions correctly. Does this help?

**Comment by aerth** [comment]
Assuming you are seeing this message: 

```
Config file already exists.
If you want to create a new config file, move or delete the existing one.
You can also set the GNUSOCIALPATH env to use multiple config files. 
Example: GNUSOCIALPATH=gnusocial.de go-quitter config
```


Instead of deleting, you may rename or specify an alternate configuration file location.

Here we use the seconf library to create a new config file: 

https://github.com/aerth/go-quitter/blob/master/cmd/go-quitter/20-config.go#L16

seconf uses a default pad in case the user supplies a short (or no) password:

https://github.com/aerth/seconf/blob/master/v1.go

When the seconf default pad changed, your config broke.

**Comment by aerth** [comment]
You can download pre-built binaries here: https://github.com/aerth/go-quitter/releases

Does this help?

**Comment by irl** [comment]
From the README:

> For automation, scripts, and cronjobs, you need to delete config file and use environmental variables instead.

I'm wondering as I'd like a post-receive hook to post when I make new blog posts, and I'd rather be using a config file than an environment variable for the password.

**Comment by irl** [comment]
I see in the changelog that this was perhaps fixed in 0.0.8, but maybe the README was left behind...
